### DISCUSSION TITLE
Initialize online-simulation-kit

### DISCUSSION THREAD

**Discussion by efficient0482** [eclipse-tractusx/sig-release#8] [General]
The team of the online-simulation-kit reached out to me. They want to add some content to their [repository](https://github.com/eclipse-tractusx/online-simulation-kit) but since it is empty they cannot create a fork nor a PR. I opened an [Issue](https://github.com/eclipse-tractusx/online-simulation-kit/issues/1#issue-1774480764) within project with the request for the project to be initialized. 
- Who do I have to tag for the initialization?
- Developer [Dominik Koeh](https://github.com/dominikoeh)

**Comment by FaGru3n** [comment]
Hi @LuisRickert ,
initialized it. see https://github.com/eclipse-tractusx/online-simulation-kit/issues/1#issuecomment-1609399337

**Reply by FaGru3n** [reply]
and you can also contact the EF-Helpdesk for repo questions within Eclipse Tractus X GitHub Org.
